package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "first-interaction-action",
-  "version": "1.1.2-zephyr",
+  "version": "1.1.3-zephyr",
```

Comment:
I am not sure how this versioning system is going to work when the upstream repository releases `1.1.2` and `1.1.3`.

If we are going to version the forked action based on the upstream version number, it should be distinguishable from the upstream version number (e.g. If the upstream version is `v1.1.1`, then something like `v1.1.1.1` or `v1.1.1-1` or `v1.1.1-zephyr-1` etc.)

Reply:
moved to `1.1.1-zephyr-1`, and will use a `v1.1.1-zephyr-1` tag when merged
fwiw the upstream repo very much looks like abandonware at this point, unfortunately.